### PR TITLE
(Feature) #801 webapp ios should start at special signin screen

### PR DESCRIPTION
**Pull request by yaroslav-fedyshyn-nordwhale**
# Description

Create endpoint to send magic code received from GoodDapp to the user by sms.
add tests for new endpoint

About #https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/801

# How Has This Been Tested?

Described in GoodDapp PR.

# Link To GoodDapp PR:
https://github.com/GoodDollar/GoodDAPP/pull/815

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request ( for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/server/send/send.sendgrid.js [hunk 1]
```diff
@@ -36,6 +36,14 @@ export const sendLinkBySMS = (to: string, link: string) => {
   return client.messages.create(plivoPhoneNumber, to, text)
 }
 
+export const sendMagicCodeBySMS = (to: string, code: string) => {
```
Comment (sirpy) [inline src/server/send/send.sendgrid.js]: this is in the wrong file. its sendgrid file, dont know why other sms message is here. its something old unused.

Comment (sirpy) [reply src/server/send/send.sendgrid.js]: we also use twilio not plivo.

**Comment by sirpy** [comment]
@yaroslav-fedyshyn-nordwhale 
We will also send 2 SMS, one with the message
"This is your easy access code for the GoodDollar wallet. Save it, you will be asked for it soon:"
then another sms with just the code

**Comment by LiavGut** [comment]
@yaroslav-fedyshyn-nordwhale 
Text changes:

Top bar - update text
Title - update text
Body text - update text

see:
https://xd.adobe.com/spec/8f259bcd-550e-4f0c-40bb-bdce2ed36fe5-f914/screen/73a383a2-4fc1-4863-8f4a-dcf33d872f64/Sign-in-iOS/

File: src/server/send/send.sendgrid.js (hunk 1 above)
Comment (yaroslav-fedyshyn-nordwhale) [reply src/server/send/send.sendgrid.js]: done

**Comment by yaroslav-fedyshyn-nordwhale** [comment]
> @yaroslav-fedyshyn-nordwhale
> We will also send 2 SMS, one with the message
> "This is your easy access code for the GoodDollar wallet. Save it, you will be asked for it soon:"
> then another sms with just the code
> 
> this is because usually when you copy an sms it copies the whole message and you can select part of it

done

**Comment by yaroslav-fedyshyn-nordwhale** [comment]
> @yaroslav-fedyshyn-nordwhale
> Text changes:
> 
> Top bar - update text
> Title - update text
> Body text - update text
> 
> see:
> https://xd.adobe.com/spec/8f259bcd-550e-4f0c-40bb-bdce2ed36fe5-f914/screen/73a383a2-4fc1-4863-8f4a-dcf33d872f64/Sign-in-iOS/

Done

**Pull request merged**